src/app/calculation/scores.py

```diff
@@ -121,24 +124,30 @@ def initialize_infra_table(cur, conn):
     cur.execute('drop table if exists infra_type_scores;')
     conn.commit()
 
-    query = f'create table  if not exists infra_type_scores (' \
+    query = f'create table if not exists infra_type_scores (' \
             f'id serial primary key, ' \
             f'infra_type text, ' \
-            f'avg_a_score numeric, ' \
-            f'avg_c_score numeric, ' \
             f'avg_p_score numeric, ' \
             f'avg_s_score numeric, ' \
             f'avg_m_p_score numeric, ' \
             f'avg_danger_score numeric, ' \
             f'count numeric, ' \
-            f'incident_count numeric, ' \
+            f'avg_a_score numeric, ' \
+            f'avg_c_score numeric, ' \
             f'avg_incident_count numeric, ' \
-            f'scary_incident_count numeric, ' \
             f'avg_scary_incident_count numeric, ' \
-            f'c_count numeric, ' \
             f'avg_c_count numeric, ' \
-            f'a_count numeric, ' \
             f'avg_a_count numeric);'
 
     cur.execute(query)
     conn.commit()
+
+
+def save_infra_type_scores(city):

```

Comment:
This is a bit schizophrenic, as we are saving the results both to the db and to csv -> We should agree on one for the final code (doesnt really matter which of the two we use in actuallity)

Reply:
was added because the database  abis not able to store results for multiple cities. Only using csv would be fine (adjusting the db would also work), but I dont want to change it anymore.

Reply:
Understandable. Might change this in a later PR if I have enough time but fine for now.